CefSharp/MouseEvent.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright © 2010-2014 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace CefSharp
+{
+    public class MouseEvent : IMouseEvent
```

Comment:
Thanks for all the great feedback Alex!
I'll run through it asap and update the PR.

On 1/8/2015 10:39:25 AM, Alex Maitland <notifications@github.com> wrote:
In CefSharp/MouseEvent.cs [https://github.com/cefsharp/CefSharp/pull/732#discussion-diff-22642543]:
> @@ -0,0 +1,17 @@ > +﻿// Copyright © 2010-2014 The CefSharp Authors. All rights reserved. > +// > +// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file. > + > +using System; > +using System.Collections.Generic; > +using System.IO; > + > +namespace CefSharp > +{ > + public class MouseEvent : IMouseEvent
Don't worry about the struct idea, we'll just keep things consistent for now.
—
Reply to this email directly or view it on GitHub [https://github.com/cefsharp/CefSharp/pull/732/files#r22642543].